packages/components/src/internal/components/entities/utils.ts

```diff
@@ -125,3 +127,17 @@ export function getUpdatedRowForParentChanges(
     });
     return updatedValues;
 }
+
+export function getUniqueIdColumnMetadata(queryInfo: QueryInfo) : Map<string, EditableColumnMetadata> {
```

Comment:
looks like some of the utility functions were static functions on the EntityInsertPanel and some here in utils.ts. moving forward, do we have a pattern for when to use each type? is it maybe based on where the function is used? or are those in the EntityInsertPanel just static to allow for jest testing?

Reply:
I left them as static functions because these are expected to be used only within this component, but I wanted a way to jest test them easily.